cypress/integration/placeholderTest.feature.js

```diff
@@ -6,4 +6,5 @@ describe('Placeholder test to set up CI', () => {
   it('is expected to visit a site', () => {
     cy.get('[data-cy=ci-test]').should('contain.text', 'App')
   });
-});
\ No newline at end of file
+});
+
```

Comment:
I suggest we delete this test now, it is no longer needed.

Reply:
Removed